api/schema/wallet.yml

```diff
@@ -0,0 +1,151 @@
+        required: true
+      responses:
+        '200':
+          description: Successful operation
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/WalletPostResponse'        
+        '405':
+          description: Invalid input
+  /wallets/{walletId}:
+    put:
+      tags:
+        - wallets
+      summary: Update an existing wallet
+      description: Update an existing wallet by Id
+      operationId: updateWallet
+      parameters:
+        - name: walletId
+          in: path
+          description: ID of wallet to return
+          required: true
+          schema:
+            type: integer
+            format: int64
+      requestBody:
+        description: Update an existent wallet
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/Wallet'
+        required: true
+      responses:
+        '200':
+          description: Successful operation
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Wallet'
+        '400':
+          description: Invalid ID supplied
+        '404':
+          description: wallet not found
+        '405':
+          description: Validation exception
+    get:
+      tags:
+        - wallets
+      summary: Find wallet by ID
+      description: Returns a single pet
+      operationId: getWalletById
+      parameters:
+        - name: walletId
+          in: path
+          description: ID of wallet to return
+          required: true
+          schema:
+            type: integer
+            format: int64
+      responses:
+        '200':
+          description: successful operation
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Wallet'    
+        '400':
+          description: Invalid ID supplied
+        '404':
+          description: Wallet not found
+    delete:
+      tags:
+        - wallets
+      summary: Deletes a wallet by ID
+      description: delete a wallet
+      operationId: deleteWallet
+      parameters:
+        - name: api_key
+          in: header
+          description: ''
+          required: false
+          schema:
+            type: string
+        - name: walletId
+          in: path
+          description: Wallet id to delete
+          required: true
+          schema:
+            type: integer
+            format: int64
+      responses:
+        '400':
+          description: Invalid wallet value
+components:
+  schemas:
+    Wallet:
+      type: string
+    WalletPost:
+      type: object
+      properties:
+        paymentInstrumentType:
+          type: string
+          enum: [ cards, paypal, bancomatpay ]
+    WalletPostResponse:
+      type: object
+      properties:
+        walletItemId:
+          type: string
+        logoutURI:
+          type: string
+          
```

Comment:
add onboardingUrl ( type string ) . This is the URI where to redirect the user in order to complete the onboarding process.